### PR TITLE
Mm posts improvements

### PR DESCRIPTION
**Pull request by roosalles**

### CODE REVIEW

**Comment by jgonzo127** [comment]
Nice additions!

File: components/posts/posts.php [hunk 1]
```diff
@@ -79,9 +80,10 @@ function mm_posts( $args ) {
 	);
 
 	// Add to our query if additional params have been passed.
-	if ( $post_id ) {
+	if ( $post_ids ) {
 
-		$query_args['p'] = $post_id;
+		$query_args['post__in'] = $post_ids;
+		$query_args['orderby']  = 'post__in';
```
Comment (BraadMartin) [inline components/posts/posts.php:86]: Rodrigo, teaching me things! Nice.

**Comment by BraadMartin** [comment]
Awesome stuff! Only thing I might consider is changing "Unlink Title" to "Link Title" and defaulting it to true. Feels a bit more natural that way. But either way great stuff.

**Comment by roosalles** [comment]
Yeah, actually, I've tried to do like that with "Link Title" but the thing
was that I couldn't find a way of having the VC checkbox checked by
default, when the component is added to the page. It's definitely much more
natural for me too, i just didn't have much time to research this checkbox
thing. Any ideas? I'd love to change and invert the logic if that's possible

File: components/posts/posts.php (hunk 1 above)
Comment (roosalles) [inline components/posts/posts.php:86]: :D

**Pull request merged**